Documentation/README.md

```diff
@@ -0,0 +1,59 @@
+# SwiftGen documentation
+
+This directory contains documentation for various topics of SwiftGen
```

Comment:
... topics related to SwiftGen, such as in-depth documentation about parsers and templates, but also articles with tips & tricks on how to best use SwiftGen in your workflow.